src/sentry/api/serializers/models/project.py

```diff
@@ -60,6 +60,24 @@
 _PROJECT_SCOPE_PREFIX = "projects:"
 
 LATEST_DEPLOYS_KEY: Final = "latestDeploys"
+UNUSED_ON_FRONTEND_FEATURES: Final = "unusedFeatures"
+
+
+# These features are not used on the frontend,
+# and add a lot of latency ~100-300ms per flag for large organizations
+# so we exclude them from the response if the unusedFeatures collapse parameter is set
+PROJECT_FEATURES_NOT_USED_ON_FRONTEND = {
```

Comment:
Maybe we could register certain features as being front-end only? This is probably OK for now but keeping this in sync with features long term isn't great

Reply:
The intention is to move away from loading all projects at app init. -- once we do that, we can likely remove this.

Reply:
also note that this only excludes ones defined in this list -- new project flags will by default be in the response.

Reply:
I think there's a case to be made for only including features in the frontend response when they're actually necessary, and including this in the feature flag registration process might make sense. We have hundreds of features at the moment, and presumably the frontend doesn't really use all of these.

Doesn't have to be in this pr, but I think it's worth considering

Reply:
for sure, definitely worth considering as we're making more changes to feature handling going forward